curp/src/server/curp_node.rs

```diff
@@ -65,6 +65,9 @@ pub(super) enum CurpError {
     /// Internal error
     #[error("internal error, {0}")]
     Internal(String),
+    /// TODO
+    #[error("internal error, {0:?}")]
```

Comment:
Please modify the error output message. It will be confusing when a user sees two errors with the totally same output message.  In addition, todo what?